pkg/kn/commands/source/sink/sink_flags.go

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2019 The Knative Authors
```

Comment:
can we move the flags to `pkg/kn/commands/flags/` ?

Reply:
Maybe sink could be moved to `pkg/kn/commands/flags/` because it will be used in several Eventing command. I will leave it to @rhuss to decide.

Reply:
I have moved flags to that folder.